generic/tests/ethtool.py

```diff
@@ -52,8 +53,12 @@ def ethtool_get(session):
         status = {}
         for f in feature_pattern.keys():
             try:
-                status[f] = re.findall(
-                    "%s: (.*)" % feature_pattern.get(f), o)[0]
+                temp = re.findall(
+                        "%s: (.*)" % feature_pattern.get(f), o)[0]
+                if temp.find("[fixed]") != -1:
+                    logging.debug("%s is fixed" % f)
+                    continue
```

Comment:
I'd suggest to keep storing the value and mark the entry as readonly.

Reply:
If the offload type is fixed, we cannot set it otherwise ethtool will return error.
Because of so, I don't think we need to store it since it will not be used.